sapl/templates/sessao/blocos_ata/materias_expediente.html

```diff
@@ -1,27 +1,42 @@
 <fieldset>
   <p align="justify">
-  {% if materia_expediente %}
+	{% if materia_expediente %}
       <strong>Matérias do Expediente: </strong> 
-      {% for m in materia_expediente %}
-            <b>{{m.numero}} - {{m.titulo}}</b>
-
-            {% if m.turno %}
-              Turno: {{m.turno}}
-            {% endif %}
-
-             Autor{{ m.autor|length|pluralize:"es" }}: {{ m.autor|join:', ' }}
-
-            {% if m.numero_protocolo %}
-               Número de Protocolo: {{ m.numero_protocolo }}
-            {% endif %}
-
-            {% if m.numero_processo %}
-               Processo: {{ m.numero_processo }}
-            {% endif %}
-
-			{{m.ementa|safe}}
-         	   {{m.resultado}} {{m.resultado_observacao}}</td>
-      {% endfor %}
+	  {% for m in materia_expediente %}
+	  	<b>{{m.numero}} - {{m.titulo}}</b>, 
+		{{m.ementa|safe}}
+		Autor{{ m.autor|length|pluralize:"es" }}: {{ m.autor|join:', ' }}, 
+		{% if m.numero_protocolo %}
+			Número de Protocolo: {{ m.numero_protocolo }}, 
+		{% endif %}
+		{% if m.numero_processo %}
+			Processo: {{ m.numero_processo }}, 
+		{% endif %}
+		{%if m.turno %}
+			Turno: {{m.turno}}, 
+		{%endif %}
+		{%if m.tipo_votacao %}
+			Tipo: {{m.tipo_votacao}}, 
+		{% endif %}
+		{% if m.voto_sim %}
+			Sim: {{ m.voto_sim }}, 
+		{% endif %}
+		{% if m.voto_nao %}
+			Não: {{ m.voto_nao }}, 
+		{% endif %}
+		{% if m.voto_abstencoes %}
+			Abstenções: {{m.voto_abstencoes}}, 
+		{% endif %}
```

Comment:
Se a votação for nominal ele vai ter votos sim, não e abstenção. Fazer um `if` pra cada situação é feio. Conserta isso.

Reply:
Realmente, mas como estou separando cada resultado por "," não encontrei uma forma mais elegante de fazer isso

Reply:
Já que vc está checando se existe `tipo_votacao` porque não coloca esses campos dentro daquele if?

Reply:
Bacana, vou fazer isso. Obrigado !